src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/OptimizedCompletionList.cs

```diff
@@ -134,7 +134,7 @@ private void WriteCompletionItem(JsonWriter writer, CompletionItem completionIte
                     writer.WriteValue(completionItem.InsertText);
                 }
 
-                if (completionItem.InsertTextFormat != InsertTextFormat.PlainText)
+                if (completionItem.InsertTextFormat != default && completionItem.InsertTextFormat != InsertTextFormat.PlainText)
```

Comment:
Unrelated optimization fix. Basically if we don't set the completion item's insert text it's default is `0` which doesn't associate with any specific enum value meaning the PlainText comparison would fail.